security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -57,6 +57,21 @@ The following advisory is available for the OpenShift Compliance Operator 0.1.61
 
 * Before this update, the `selinux_confinement_of_daemons` rule failed running on the kubelet because of the permissions necessary for the kubelet to run. With this update, the `selinux_confinement_of_daemons` rule is disabled. (link:https://issues.redhat.com/browse/OCPBUGS-6968[*OCPBUGS-6968*])
 
+[id="compliance-operator-0-1-61-known-issue"]
+=== Known issue
```

Comment:
Content itself looks good. 

Question more to satiate my own curiosity: what is the known issue?  That the variable is set to 10 when it used to be 5?

Reply:
Correct, the change in default values may be unexpected by customers.

Reply:
@Vincent056 customers won't need this workaround when https://github.com/ComplianceAsCode/content/pull/10464 merges, correct?

Reply:
No, they will not need that workaround with this patch